userspace/libscap/scap_vtable.c

```diff
@@ -0,0 +1,148 @@
+#include <stdio.h>
+
+#include "scap.h"
+#include "scap-int.h"
+
+int32_t scap_next(scap_t* handle, OUT scap_evt** pevent, OUT uint16_t* pcpuid)
+{
+	struct scap_engine_handle eh = {
```

Comment:
Hopefully this will go away with subsequent patches, right?

Reply:
I don't feel it's worth it (the vtable wrappers won't all be straight 1:1 calls into the vtable)

> perhaps using m4

Ah, now I noticed you're clearly joking :D carry on then :D